src/yunohost/utils/error.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+
+""" License
+
+    Copyright (C) 2018 YUNOHOST.ORG
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program; if not, see http://www.gnu.org/licenses
+
+"""
+
+from moulinette.core import MoulinetteError
+from moulinette import m18n
+
+class YunohostError(MoulinetteError):
```

Comment:
What is the idea of doing a specific class for YunoHost errors? I don't see any justification nowhere and that should be in a comment for when we'll ask ourself this not obvious question in 6 month or more.

Reply:
The idea is that m18n.g got factorized in MoulinetteError. But in the context of YunoHost, we shall use m18n.n, hence the introduction of YunohostError.

(I added a comment https://github.com/YunoHost/yunohost/pull/574/commits/46972788b2f4c3b3ac79e2d2fb9b8dd6a3834148 )